.commitlintrc

```diff
@@ -0,0 +1,8 @@
+{
+  "extends": [
+    '@commitlint/config-conventional'
+  ],
+  "rules": {
+    "scope-enum": [ 2, 'always', [ 'release', 'eslint', 'eslint-saga' ] ],
```

Comment:
maybe we should add scopes as node, react, react-native and eslint? 🤔

Reply:
I though of making these scopes for packages (eslint, eslint-redux-saga). But maybe you're right, node, react, react-native, redux-saga would make more sense?

Reply:
We can omit eslint because it is the general commit without the scope

Reply:
Yes, great thinking 👍